apps/homescreen/test/unit/pages_test.js

```diff
@@ -298,6 +304,7 @@ suite('Pages', () => {
       storeGetStub.restore();
       storePutStub.restore();
       pages.selectedCard = null;
+      callback = null;
```

Comment:
Needed to avoid `done` being called from previous test when running the new test below.